src/module/actor/mixins/actor-conditions.js

```diff
@@ -11,6 +17,12 @@ export const ActorConditionsMixin = (superclass) => class extends superclass {
         return (conditionItem !== undefined);
     }
 
+    /**
+     * Get a condition Item from the actor.
+     * @param {String} conditionName The name of the condition. Must match any key from config.js SFRPG.statusEffectIconMapping. Case insensitive.
```

Comment:
The conditionName is suggested to be case insensitive in your comments, but it is case sensitive.

Reply:
Same as above